### ISSUE TITLE
dependency issue on DotNext.Net.Cluster 4.12.2

### ISSUE THREAD

**Issue by freddyrios** [dotnet/dotNext#173]
Building with version 4.12.2 of DotNext.Net.Cluster fails with:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1705	Assembly 'DotNext.Net.Cluster' with identity 'DotNext.Net.Cluster, Version=4.12.2.0, Culture=neutral, PublicKeyToken=c8827b44e18fe572' uses 'DotNext, Version=4.12.2.0, Culture=neutral, PublicKeyToken=c8827b44e18fe572' which has a higher version than referenced assembly 'DotNext' with identity 'DotNext, Version=4.12.0.0, Culture=neutral, PublicKeyToken=c8827b44e18fe572'	LoopComponents	C:\Users\fredd\source\repos\CA_LoopCode\LoopComponents\CSC	1	Active

```
It seems that even though 4.12.2 is only released for DotNext.Net.Cluster, the build still uses the 4.12.2 for the dependencies.

**Comment by sakno** [comment]
Add DotNext core library with 4.12.2 to your csproj explicitly.

**Comment by sakno** [comment]
To make it _just works_, I can publish all upgraded dependencies.

**Comment by sakno** [comment]
Fixed in 4.12.3.

**Comment by freddyrios** [comment]
great, thanks!
